src/Altinn.Apps/AppFrontend/react/receipt/webpack.config.production.js

```diff
@@ -23,5 +23,10 @@ module.exports = {
       },
     ],
   },
-  plugins: [...commonConfig.plugins, new ForkTsCheckerWebpackPlugin()],
+  plugins: [
+    ...commonConfig.plugins,
+    new ForkTsCheckerWebpackPlugin({
+      async: false,
```

Comment:
Out of curiosity, why don't we run this async for prod?

Reply:
After looking at this again, I dont think we actually need to set it at all. It defaults to `true` for development and `false` for production builds, and additionaly it is only used in `watch` mode.

I think I'll just remove these configs since we anyway are using the defaults.

As for why not use async in production builds, I'm not entirely sure, but I guess it wouldnt make much difference? It would have to wait until webpack compilation is done anyway.

Reply:
Ah, the docs on the GitHub repo were a little better than on the npm page. I thought we had actively made this choice.
Thanks for the clarification 👍

Reply:
I have removed this config option now. That also made the config identical between dev and prod, so the entire webpack config got a little bit simpler :)